### DISCUSSION TITLE
Can't Uploded Student Verification Documents

### DISCUSSION THREAD

**Discussion by hirenyadav** [community/community#35988] [GitHub Education]
Hello All 

I am Student , I Want Student Developer Pack For Learning Something New But I Can't Uploding Images Or take From Camera 
Just Loading And Loading Anyone Have This Solution Please Help me

Thanks in advanced.

**Comment by jostasik** [comment]
Hmmm are you using the correct image format? 

Maybe the issue is being caused by a browser extension or a VPN connection being weird or something? I would first re-try it in an incognito window to narrow the problem down to GitHub itself.

**Reply by hirenyadav** [reply]
Hello jostasik,

Uploding Images not working and No Vpn Used 

I Also tried to incognito window but still facing this issue...
